src/OrleansRuntime/Catalog/ActivationData.cs

```diff
@@ -362,6 +348,10 @@ public GrainId Identity
 
         public IDictionary<object, object> Items { get; private set; }
 
+        public IGrainLifeCycle LifeCycle => FullLifeCycle;
+
+        internal GrainLifecycle FullLifeCycle { get; }
```

Comment:
still a few of these left

---
In reply to: [126530614](https://github.com/dotnet/orleans/pull/3208#discussion_r126530614) [](ancestors = 126530614)